editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -524,6 +524,7 @@ bool TileMapEditorTilesPlugin::forward_canvas_gui_input(const Ref<InputEvent> &p
 	}
 
 	if (CanvasItemEditor::get_singleton()->get_current_tool() != CanvasItemEditor::TOOL_SELECT) {
+		drag_type = DRAG_TYPE_NONE;
```

Comment:
You should use `_stop_dragging();` instead. This functions makes sure nothing is in an inconsistent state once dragging is done.

Reply:
this should be fixed now